src/app/users/postEditProfile.js

```diff
@@ -116,8 +116,19 @@ const postEditProfile = async (req, res) => {
     }
   }
 
-  await updateUser(uid, req.body.firstName, req.body.lastName, req.id);
-  await updateUserIndex(uid, req.body.firstName, req.body.lastName, req.id);
+  const newName = {
```

Comment:
Super minor, but this only gets used in the if statement below.  I'd move this variable inside the if